tests/test_optimization/test_response_mime_type.py

```
@@ -0,0 +1,92 @@
+
+    response = litellm.stream_chunk_builder(chunks, messages=[{"role": "user", "content": prompt}])
+    assert response is not None, "stream_chunk_builder returned None"
+
+    content = response.choices[0].message.content
+    if not content:
+        # Apply the same delta.content fallback the framework uses
+        content = "".join(
+            (c.choices[0].delta.content or "") for c in chunks if c.choices
+        )
+
+    print(f"\n--- RESPONSE WITH response_format ---")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)